### DISCUSSION TITLE
Problems with connecting to azure instance

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#10138] [Unsupported Versions] [answered]
Im having some problems getting SO to work on azure. Im using the image from the marketplace and following the installer. But when im trying to connect to SOC website, im only redirected to the internal MS Ip, so nothing is showing.
Is there any trick to make this work?

**Comment by InfosecGoon** [comment, marked as answer]
It sounds like you set up the Manager to use an IP address for SOC -- what is the output from this command?

` salt-call pillar.get global:url_base`
